slides/week5/week5.org

```diff
@@ -0,0 +1,80 @@
+#+TITLE: Week 5
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
+
+* What are we doing today?
+- Classes in C++
+- Handling merge conflicts in =git=
+
+* What are classes?
+- Definitions for objects, i.e. what data and methods they have
+- An object is called an *instance* of a class
+
+* A simple example
+#+BEGIN_NOTES
+A class can have some data and some methods, which can either be public or private. Private means only visible to the class itself, public means visible to any other file. Drop to a terminal and use/modify this class.
+#+END_NOTES
+#+BEGIN_SRC c
```

Comment:
These should be `C++` not `c` (they produce compiler errors, and don't recognize C++ only keywords in highlighting).